### PR TITLE
Refactor: update feature plugin namespaces to match GiveWP

### PR DESCRIPTION
**Pull request by jonwaldstein**
🛑 Depends on: https://github.com/impress-org/givewp/pull/6817 & https://github.com/impress-org/givewp/pull/6820

## Description

This PR aims to make the folder structure and namespaces match what's in GiveWP core.  Along with that, an effort to rename all "Next Gen" terminology to something more generic and/or "v3".

For the time being, the "next gen" test gateways will remain their same names.  Eventually we'll have a core test gateway that these will be merged into.

## Affects

Everything...

May or may not need to purge any old forms and start from scratch.

## Visuals

N/A

## Testing Instructions

~~🛑 Make sure GiveWP is set to this branch: https://github.com/impress-org/givewp/pull/6817~~
~~🛑 Make sure GiveWP is set to this branch: https://github.com/impress-org/givewp/pull/6820~~

Then, create some v3 forms and test everything out, including making donations with gateways.

## Pre-review Checklist

-   [ ] Acceptance criteria satisfied and marked in related issue
-   [ ] Relevant `@unreleased` tags included in DocBlocks
-   [ ] Includes unit tests
-   [ ] Reviewed by the designer (if follows a design)
-   [ ] [Self Review](https://give.gitbook.io/development-manual/devops/github/code-reviews#self-review) of code and UX completed

### CODE REVIEW

**Comment by jonwaldstein** [comment]
@pauloiankoski @kjohnson this is ready for review

**Comment by jonwaldstein** [comment]
@JasonTheAdams this is ready for review.

File: .github/workflows/unit-tests.yml [hunk 1]
```diff
@@ -11,4 +11,4 @@ jobs:
         uses: impress-org/givewp-github-actions/.github/workflows/addon-tests.yml@master
         with:
             addon_slug: givewp-next-gen
-            givewp_branch: develop
+            givewp_branch: feautre/show-in-admin
```
Comment (JasonTheAdams) [inline .github/workflows/unit-tests.yml:14]: @jonwaldstein Are we going to set this back after this is merged?

**Review by JasonTheAdams — APPROVED** [review]
Thanks for updating the domains as we discussed! 👍

File: .github/workflows/unit-tests.yml (hunk 1 above)
Comment (jonwaldstein) [reply .github/workflows/unit-tests.yml:14]: yesszir

**Pull request merged**